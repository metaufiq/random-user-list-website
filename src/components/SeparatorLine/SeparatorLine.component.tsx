const SeparatorLine = ()=>{
  return(<div className="border-t-gray-100 border-t-2"/>)
}

export default SeparatorLine;