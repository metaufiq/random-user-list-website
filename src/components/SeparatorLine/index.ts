import SeparatorLine from "./SeparatorLine.component";

export default SeparatorLine;