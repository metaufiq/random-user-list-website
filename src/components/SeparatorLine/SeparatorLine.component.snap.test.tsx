import { assertSnapshots } from "../../../test";
import SeparatorLine from ".";

describe('SeparatorLineSnapshot', () => {

  const configs = [
    {
      desc: 'should render normal SeparatorLine'
    }
  ];

  assertSnapshots(SeparatorLine, configs);
});
