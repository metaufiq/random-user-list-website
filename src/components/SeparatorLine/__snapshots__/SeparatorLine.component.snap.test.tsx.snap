// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeparatorLineSnapshot should render normal SeparatorLine 1`] = `
<div
  className="border-t-gray-100 border-t-2"
/>
`;
